core/src/main/scala/org/locationtech/rasterframes/util/package.scala

```diff
@@ -203,6 +203,25 @@ package object util {
         .mkString("| ", " |\n| ", " |")
       header + body
     }
+
+    def toHTML(numRows: Int = 5, truncate: Boolean = false): String = {
+      import df.sqlContext.implicits._
+      val cols = df.columns
+      val header = "<thead>\n" + cols.mkString("<tr><th>", "</th><th>", "</th></tr>\n") + "</thead>\n"
+      val stringifiers = cols
+        .map(c => s"`$c`")
+        .map(c => df.col(c).cast(StringType))
+        .map(c => if (truncate) substring(c, 1, 40) else c)
+      val cat = concat_ws("</td><td>", stringifiers: _*)
+      val body = df
+        .select(cat).limit(numRows)
```

Comment:
So we would really like to have some indication on the output that truncation has happened. Even if it is just like teh spark show message `"Showing only top 5 rows."` that would be better....

Obviously we could have a case where we print that and there is only 1 row or the DF is empty.

Possibly collect then check if length is < `numRows` to adjust appearance of message?

Reply:
Done :-)